cis_supply_chain_v100/docs/cis_supply_chain_v100_overview.md

```diff
@@ -0,0 +1,29 @@
+To obtain the latest version of the official guide, please visit http://benchmarks.cisecurity.org.
```

Comment:
Does the link contain the latest CIS supply chain document?

Reply:
@misraved in general people need to login and check.